autoscheduler/scraper/tests/api_tests.py

```diff
@@ -0,0 +1,253 @@
+        second_end = time(10)
+        meeting_days = [True] * 7
+        expected = {
+            'crn': 12345,
+            'instructor_gpa': None,
+            'instructor_name': 'Akash Tyagi',
+            'honors_only': False,
+            'meetings': {
+                '10': {
+                    'days': meeting_days,
+                    'start_time': first_start,
+                    'end_time': first_end,
+                    'type': 'LEC',
+                },
+                '11': {
+                    'days': meeting_days,
+                    'start_time': second_start,
+                    'end_time': second_end,
+                    'type': 'LEC',
+                },
+            },
+            'section_num': '501',
+            'web_only': False,
+        }
+
+        # Act
+        serializer = SectionSerializer(self.sections[0])
+
+        # Assert
+        self.assertEqual(expected, serializer.data)
+
+    def test_api_sections_gives_valid_response(self):
+        """ Tests that /api/sections?&dept=CSCE&course_num=310term=201931 gives the
+            correct output
+        """
+        # Arrange
+        first_start = time(11, 30)
+        first_end = time(12, 20)
+        second_start = time(9, 10)
+        second_end = time(10)
+        meeting_days_true = [True] * 7
+        meeting_days_false = [False] * 7
+        expected = {
+            '1': {
+                'crn': 12345,
+                'instructor_gpa': None,
+                'instructor_name': 'Akash Tyagi',
+                'honors_only': False,
+                'meetings': {
+                    '10': {
+                        'days': meeting_days_true,
+                        'start_time': first_start,
+                        'end_time': first_end,
+                        'type': 'LEC',
+                    },
+                    '11': {
+                        'days': meeting_days_true,
+                        'start_time': second_start,
+                        'end_time': second_end,
+                        'type': 'LEC',
+                    },
+                },
+                'section_num': '501',
+                'web_only': False,
+            },
+            '2': {
+                'crn': 12346,
+                'instructor_gpa': None,
+                'instructor_name': 'John Moore',
+                'honors_only': False,
+                'meetings': {
+                    '20': {
+                        'days': meeting_days_true,
+                        'start_time': first_start,
+                        'end_time': first_end,
+                        'type': 'LEC',
+                    },
+                    '21': {
+                        'days': meeting_days_false,
+                        'start_time': second_start,
+                        'end_time': second_end,
+                        'type': 'LAB',
+                    },
+                },
+                'section_num': '502',
+                'web_only': False,
```

Comment:
I thought it looked nicer and using ids as properties for sections/meetings is still pretty intuitive, but if you prefer, we can do the following instead (this new version is how django rest framework formats list views by default). I'd like if @firejake308 could say which he prefers as well, I get where you're coming from and it would be a very easy change to make:
```json
[
    {
        "id": "10923201931",
        "crn": 10923,
        "instructor_gpa": null,
        "instructor_name": "John M. Moore",
        "honors_only": null,
        "meetings": [
            {
                "id": "4683040",
                "days": [
                    true,
                    false,
                    true,
                    false,
                    true,
                    false,
                    false
                ],
                "start_time": "11:30:00",
                "end_time": "12:20:00",
                "type": "LEC"
            },
            {
                "id": "4683041",
                "days": [
                    true,
                    false,
                    true,
                    false,
                    false,
                    false,
                    false
                ],
                "start_time": "15:00:00",
                "end_time": "15:50:00",
                "type": "LAB"
            },
        ],
        "section_num": "503",
        "web_only": null
    },
    {
        "id": "39493201931",
        "crn": 39493,
        "instructor_gpa": null,
        "instructor_name": "John M. Moore",
        "honors_only": null,
        "meetings": [
            {
                "id": "4911320",
                "days": [
                    true,
                    false,
                    true,
                    false,
                    true,
                    false,
                    false
                ],
                "start_time": "11:30:00",
                "end_time": "12:20:00",
                "type": "LEC"
            },
            {
                "id": "4911321",
                "days": [
                    true,
                    false,
                    true,
                    false,
                    false,
                    false,
                    false
                ],
                "start_time": "19:00:00",
                "end_time": "19:50:00",
                "type": "LAB"
            }
        ],
        "section_num": "504",
        "web_only": null
    },
]
```